src/components/sortableGridView/SortableGridItemAnimationWrapper.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
```

Comment:
I think we can simplify the name and call this component `SortableGridItem`

Reply:
np